### PR TITLE
Improved wording GSI-850

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
Changed wording to some texts and look of some buttons
Removed logout button from profile page

### CODE REVIEW

File: src/components/header/loginButton.tsx [hunk 1]
```diff
@@ -179,15 +179,20 @@ const LoginButton = () => {
...
+                      <Link
+                        to="https://profile.aai.lifescience-ri.eu/profile"
                         target="_blank"
                         rel="noreferrer"
                       >
-                        Manage your
-                        <br />
-                        LS Login account
-                      </a>
+                        <Button
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/components/header/loginButton.tsx:188]: Done

File: src/components/profile/profile.tsx [hunk 2]
```diff
@@ -499,7 +499,9 @@ const Profile = () => {
           <Card.Body>
             <div>
               <p className="mb-1">
-                You have the following pending access requests:
+                {pendingUserRequests && pendingUserRequests?.length > 0
+                  ? "You have the following pending access requests:"
+                  : "You have no pending access requests:"}
               </p>
               <ul>
                 {pendingUserRequests?.map((x) => (
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/components/profile/profile.tsx]: Fixed

File: src/components/profile/profile.tsx [hunk 3]
```diff
@@ -266,12 +266,14 @@ const Profile = () => {
     content = (
       <div>
         <style lang="css">{wideTooltips}</style>
-        <h3 style={{ margin: "1em 0" }}>Welcome, {user.full_name}!</h3>
+        <h3 style={{ margin: "1em 0" }}>{user.full_name}</h3>
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/components/profile/profile.tsx]: I did the latter. Is that ok?

File: src/components/header/loginButton.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Cito) [reply src/components/header/loginButton.tsx:188]: Perfect.

File: src/components/profile/profile.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Cito) [reply src/components/profile/profile.tsx]: Looks better to me. We can discuss the terminology "profile" vs "account" separately.

**Review by Cito — APPROVED** [review]

**Pull request merged**